governance.md

```diff
@@ -59,260 +70,170 @@ The foundations of Project governance are:
+Subsequent changes to the membership of the Steering Committee will occur towards the
+end of each calendar year.  By November 30 of each year, each member of the Steering
+Committee will be asked if they would like to continue in that role. If any member
+decides to step down from the Steering Committee, new volunteers will be solicited from
+the group of stewards by the Steering Committee.  Those new volunteers will then be on a
+slate for an election that will occur by December 15 of that year.  For each year after
+the initial election, the current Steering Committee members plus any new volunteers on
+the slate will be elected via approval voting, with the top 5 candidates elected as
+Steering Committee members for the subsequent year.
+
+## Teams
+
+Each Team has defined responsibilities for different aspects of the project.  As a
+general rule, an Individual Contributor can be nominated by a member of a Team to become
+a member of that Team, and the Team must unanimously agree to admitting that person to
+the Team, since every Team member becomes a Steward of the Project. If a member of a
+Team is inactive for more than one year, the Team may unanimously decide to remove that
+person from the Team. Each Team will maintain its own private mailing list if the Team
+deems it to be necessary. There will also be a mailing list consisting of all Stewards
+of all Teams.
+
+Each Team has specific responsibilities as well as Permissions authorities, as defined
+below. In addition, specific criteria are used by Team Members to guide decisions on
+adding new Members to the Team.
+
+The description of each Team can be found in https://github.com/pandas-dev/pandas-governance/teams.md .
+
+**THE DETAILS OF THE TEAMS IN THAT DOCUMENT ARE IN A SEPARATE PULL REQUEST**
 
 ### Conflict of interest
 
-It is expected that the BDFL and Core Team Members will be employed at a wide
+It is expected that all Stewards will be employed at a wide
 range of companies, universities and non-profit organizations. Because of this,
-it is possible that Members will have conflict of interests. Such conflict of
+it is possible that Stewards will have conflict of interests. Such conflict of
 interests include, but are not limited to:
 
 -   Financial interests, such as investments, employment or contracting work,
     outside of The Project that may influence their work on The Project.
 -   Access to proprietary information of their employer that could potentially
     leak into their work with the Project.
 
-All members of the Core Team, BDFL included, shall disclose to the rest of the
-Core Team any conflict of interest they may have. Members with a conflict of
-interest in a particular issue may participate in Core Team discussions on that
-issue, but must recuse themselves from voting on the issue. If the BDFL has
-recused his/herself for a particular decision, they will appoint a substitute
-BDFL for that decision.
+All Stewards shall disclose to the Steering Committee
```

Comment:
What happens if a conflict of interest is not disclosed and later discovered?

Reply:
Our current governance policy doesn't cover this case.  Open to suggestions as to how to handle the case you bring up.

Reply:
Maybe a good default is to say that they are then removed from the team and have to be voted back in? I don't know if that is too extreme but it at least signs the rest of the committee up to sign off on the new disclosure not being problematic

Reply:
Have we done this to date w.r.t conflicts of interest? What kind of things are in scope? Core Developer on another project or works in a company that uses pandas heavily/has an agenda?

Reply:
We borrowed this text from numpy:  https://numpy.org/doc/stable/dev/governance/governance.html#conflict-of-interest .  We haven't had a policy of how to deal with a lack of disclosure.

We do list the potential conflicts as follows:

Such conflict of interests include, but are not limited to:

- Financial interests, such as investments, employment or contracting work, outside of The Project that may influence their work on The Project.
-  Access to proprietary information of their employer that could potentially leak into their work with the Project.

Reply:
It is hard to say what will happen if a conflict of interest is not disclosed.

We'll add text like this:  "If a conflict of interest is not disclosed and later uncovered, it will be left to the Steering Committee to decide how to handle the lack of disclosure on a case-by-case basis"

Reply:
made above change in next revision.